packages/clay-dropdown/src/ClayDropdownBase.js

```diff
@@ -72,6 +72,7 @@ ClayDropdownBase.STATE = {
   button: Config.shapeOf({
     label: Config.string().required(),
     style: Config.oneOf(['primary', 'secondary']).value('primary'),
+    type: Config.oneOf(['button', 'reset', 'submit']),
```

Comment:
Depends on if we want it to have a default value or not. In this case default can be `button`, but there can be cases where we want default to be undefined.

Reply:
In the case of the `button` param in `ClayDropdown` I wouldn't add defaults since we're passing entire object to `ClayDropdownBase` and we can't define that default in soy, so is `ClayDropdownBase` the one responsible of that.